client/src/components/registration.js

```diff
@@ -0,0 +1,48 @@
+import { Component } from 'preact'
+import withStyles from './jss'
+
+export default withStyles({
+  root: {
+    padding: '25px'
+  },
+  submit: {
+    'margin-top': '25px'
+  }
+}, class Register extends Component {
+  componentDidMount () {
+    document.title = 'Registration'
```

Comment:
Should we append the CTF name to the title? Maybe `Registration | redpwnCTF 2020`.

Reply:
Added to configuration file in `client/config`